### PR TITLE
Add virsh command of managedsave

### PR DESCRIPTION
**Pull request by chloerh**
- managedsave-dumpxml
- managedsave-define
- managedsave-edit

Signed-off-by: haizhao <haizhao@redhat.com>

### CODE REVIEW

File: virttest/virsh.py [hunk 1]
```diff
@@ -985,6 +985,40 @@ def managedsave_remove(name, **dargs):
     return command("managedsave-remove --domain %s" % name, **dargs)
 
 
+def managedsave_dumpxml(name, options="", **dargs):
+    """
+    Dump XML of domain information for a managed save state file.
+
+    :param name: Name of domain to dump
+    :param options: options: options to pass to list command
```
Comment (sathnaga) [inline virttest/virsh.py]: `:param options: options:` --> `param options:`

File: virttest/virsh.py [hunk 2]
```diff
@@ -985,6 +985,40 @@ def managedsave_remove(name, **dargs):
...
+    """
+    return command("managedsave-dumpxml --domain %s %s" % (name, options), **dargs)
+
+
+def managedsave_edit(name, options="", **dargs):
+    """
+    Edit the domain XML associated with the managed save state file.
+
+    :param name: Name of domain to edit
+    :param options: options: options to pass to list command
```
Comment (sathnaga) [inline virttest/virsh.py]: same here

File: virttest/virsh.py [hunk 3]
```diff
@@ -985,6 +985,40 @@ def managedsave_remove(name, **dargs):
...
+    return command("managedsave-edit --domain %s %s" % (name, options), **dargs)
+
+
+def managedsave_define(name, xml_path, options="", **dargs):
+    """
+    Replace the domain XML associated with a managed save state file.
+
+    :param name: Name of domain to define
+    :param xml_path: Path of xml file to be defined
+    :param options: options: options to pass to list command
```
Comment (sathnaga) [inline virttest/virsh.py]: same here

**Comment by sathnaga** [comment]
LGTM, apart from minor doc str comments above

File: virttest/virsh.py (hunk 3 above)
Comment (chloerh) [reply virttest/virsh.py]: Updated.

**Pull request merged**